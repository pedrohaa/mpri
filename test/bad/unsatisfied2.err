File "bad/unsatisfied2.f", line 17, characters 6-23:
This type equation should but does not hold:
a = c
Hypotheses:
_ = a
__1_ = b
