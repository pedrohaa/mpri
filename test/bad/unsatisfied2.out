Esquema: forall a b. {} -> eq a b where a = b
eq int bool
